backend/tracim_backend/views/search_api/schemas.py

```diff
@@ -12,17 +11,12 @@
 from tracim_backend.lib.utils.utils import DATETIME_FORMAT
 from tracim_backend.views.core_api.schemas import ContentDigestSchema
 from tracim_backend.views.core_api.schemas import ContentMinimalSchema
-from tracim_backend.views.core_api.schemas import EnumField
+from tracim_backend.views.core_api.schemas import RestrictedStringField
 from tracim_backend.views.core_api.schemas import StringList
 from tracim_backend.views.core_api.schemas import StrippedString
 from tracim_backend.views.core_api.schemas import UserInfoContentAbstractSchema
 
-
-class SearchContentField(Enum):
-    LABEL = "label"
-    RAW_CONTENT = "raw_content"
-    COMMENT = "comment"
-    DESCRIPTION = "description"
+search_content_field_values = ["label", "raw_content", "comment", "description"]
```

Comment:
What the reason behind removing enum here ?

Reply:
Yes, the strings where deserialized as enum fields, that I had to convert back to string anyway so it was a bit annoying.